pandas/tests/reshape/test_cut.py

```diff
@@ -725,3 +725,14 @@ def test_cut_with_nonexact_categorical_indices():
     )
 
     tm.assert_frame_equal(expected, result)
+
+
+def test_cut_with_timestamp_tuple_labels():
+    # GH 40661
+    data = [2, 4, 6]
+    bins = [1, 3, 5, 7]
```

Comment:
You don't have to define data and bins, you are using it only once. Can directly pass it into cut

Reply:
Yes definitely. Thank you :+1: